.circleci/config.yml

```diff
@@ -124,6 +124,8 @@ jobs:
         run:
           name: Install integration test dependencies
           command: |
+            sudo apt-get update -y
```

Comment:
I know this is traditional when you do it on your desktop, but is it necessary here?
Using whatever metadata is on the image seems preferable, for consistency of test runs.  If that works.

Reply:
Yes! The apt-get cache seems to be completely empty when the vm spins up so if you skip this it says "cannot find package conntrack" etc.

Reply:
Most of the time in CircleCI. I have not actually tried it in this case. Lets give it a quick test..

Reply:
It did not work, gotta update

Reply:
Ah, thanks for trying.